KenticoCloudDotNetGenerators.Tests/CodeGeneratorTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.CodeAnalysis;
+using NUnit.Framework;
+using System.IO;
+using System.Linq;
+
+namespace KenticoCloudDotNetGenerators.Tests
+{
+    [TestFixture]
+    public class CodeGeneratorTests
+    {
+        [TestCase]
+        public void IntegrationTest()
```

Comment:
We should add way to test DeliveryClient without making actual HTTP requests to the delivery API. That project I'm referencing in that test is my Sandbox project where I do all kinds of experiments, so I can't ensure that content type will stay there forever.